.circleci/config.yml

```diff
@@ -7,6 +7,11 @@ executors:
       MAJOR_VERSION: 1.
     docker:
       - image: circleci/buildpack-deps:stretch
+tagged_build_filters: &tagged_build_filters
```

Comment:
This is pretty cool. Is this `type` defined in CircleCI? Wasn't able to find any documentation.

Reply:
@RYadav95, I have already tested that pattern on some others - added with @mindjiver's suggestion

Reply:
Yeah!! I was just wondering where you guys got this idea from. If I can read a little more about this for my own knowledge

Reply:
I'll hook you up with some docs tomorrow.